http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/message/HTTPConnectorUtil.java

```diff
@@ -92,7 +95,32 @@ public static ListenerConfiguration buildListenerConfig(String id, Map<String, S
             config.setKeyStoreFile(properties.get(Constants.HTTP_KEY_STORE_FILE));
             config.setKeyStorePass(properties.get(Constants.HTTP_KEY_STORE_PASS));
             config.setCertPass(properties.get(Constants.HTTP_CERT_PASS));
-            //todo fill truststore stuff
+            if (properties.get(Constants.HTTP_TRUST_STORE_FILE) != null) {
+                config.setTrustStoreFile(properties.get(Constants.HTTP_TRUST_STORE_FILE));
+            }
+            if (properties.get(Constants.HTTP_TRUST_STORE_PASS) != null) {
+                config.setTrustStorePass(properties.get(Constants.HTTP_TRUST_STORE_PASS));
+            }
+            if (properties.get(Constants.SSL_VERIFY_CLIENT) != null) {
+                config.setVerifyClient(properties.get(Constants.SSL_VERIFY_CLIENT));
+            }
+            List<Parameter> serverParams = new ArrayList<>();
+            if (properties.get(Constants.SERVER_SUPPORT_HTTPS_PROTOCOLS) != null) {
```

Comment:
Constants.SERVER_SUPPORT_HTTPS_PROTOCOLS 

what does it mean ?

Reply:
Changed into Constants.SERVER_SUPPORT_SSL_PROTOCOLS